components/style/values/specified/length.rs

```diff
@@ -337,6 +354,24 @@ impl Mul<CSSFloat> for AbsoluteLength {
     }
 }
 
+impl Add<AbsoluteLength> for AbsoluteLength {
```

Comment:
Why do we need this? Can't we just derive `ComputeSquaredDistance` with a `#[distance(fallback)]` attribute?

Reply:
Heh